### PR TITLE
Adding section for new feature - useQueryString

### PR DESCRIPTION
**Pull request by joeynmq**
### Proposed changes
The Engineering team has just released a new feature for Querystring API - useQueryString which allows customers to optionally add constraints to how parameters from query string are processed. Constraints include but not limited to, maximum and/or minimum length.

Currently, it is available on the NPM version but not on the CDN version.

### Merge timing
ASAP once approved

### Related issues (optional)
- https://github.com/segmentio/analytics-next/pull/774

### CODE REVIEW

File: src/connections/sources/catalog/libraries/website/javascript/querystring.md [hunk 1]
```diff
@@ -30,3 +30,27 @@ analytics.user().anonymousId('abc123');
...
+## How can I turn off the query string processing or add a condition to when it should be tracked?
+
+Using `useQueryString` option enable you to control the query parameters behaviour. For example, you can entirely disable query string processing by setting `useQueryString` to `false`:
+```
+analytics.load('<WRITE_KEY>', {
+  useQueryString: false
+})
+```
+It is also possible to keep query string processing on, but enforce validation rules. For example:
+```
```
Comment (forstisabella) [inline src/connections/sources/catalog/libraries/website/javascript/querystring.md:44]: Adding the `js` to the code snippet formatting helps the syntax highlighter mark up the code appropriately!
```suggestion
```js
```

File: src/connections/sources/catalog/libraries/website/javascript/querystring.md [hunk 2]
```diff
@@ -30,3 +30,27 @@ analytics.user().anonymousId('abc123');
 ```
 
 Each trigger parameter is optional. You can pass up to **one of each trigger parameter** as shown in the example above.
+
+
+## How can I turn off the query string processing or add a condition to when it should be tracked?
+
+Using `useQueryString` option enable you to control the query parameters behaviour. For example, you can entirely disable query string processing by setting `useQueryString` to `false`:
+```
```
Comment (forstisabella) [inline src/connections/sources/catalog/libraries/website/javascript/querystring.md:38]:
```suggestion
```js
```
```

File: src/connections/sources/catalog/libraries/website/javascript/querystring.md [hunk 3]
```diff
@@ -30,3 +30,27 @@ analytics.user().anonymousId('abc123');
 ```
 
 Each trigger parameter is optional. You can pass up to **one of each trigger parameter** as shown in the example above.
+
+
+## How can I turn off the query string processing or add a condition to when it should be tracked?
```
Comment (forstisabella) [inline src/connections/sources/catalog/libraries/website/javascript/querystring.md:35]:
```suggestion
## How can I control query string processing?
```

File: src/connections/sources/catalog/libraries/website/javascript/querystring.md [hunk 4]
```diff
@@ -30,3 +30,27 @@ analytics.user().anonymousId('abc123');
...
+
+Using `useQueryString` option enable you to control the query parameters behaviour. For example, you can entirely disable query string processing by setting `useQueryString` to `false`:
```
Comment (forstisabella) [inline src/connections/sources/catalog/libraries/website/javascript/querystring.md:37]:
```suggestion
The `useQueryString` option allows you to control the behavior of the query parameters. For example, you can entirely disable query string processing by setting `useQueryString` to `false`:
```

File: src/connections/sources/catalog/libraries/website/javascript/querystring.md [hunk 5]
```diff
@@ -30,3 +30,27 @@ analytics.user().anonymousId('abc123');
...
+```
+analytics.load('<WRITE_KEY>', {
+  useQueryString: false
+})
+```
+It is also possible to keep query string processing on, but enforce validation rules. For example:
```
Comment (forstisabella) [inline src/connections/sources/catalog/libraries/website/javascript/querystring.md:43]:
```suggestion
You can also keep query string processing on, but enforce validation rules. For example:
```

**Review by forstisabella — CHANGES_REQUESTED** [review]

File: src/connections/sources/catalog/libraries/website/javascript/querystring.md [hunk 6]
```diff
@@ -30,3 +30,27 @@ analytics.user().anonymousId('abc123');
...
+    // set a pattern for anonymous id 
+    aid: /([A-Z]{10})/,
+    // set a pattern for user id
+    uid: /([A-Z]{6})/
+  }
+})
+```
+
+> info ""
+> Currently, the `useQueryString` option is **only** available when you load analytics.js through the [NPM package](https://www.npmjs.com/package/@segment/analytics-next){:target="_blank"}.
```
Comment (forstisabella) [inline src/connections/sources/catalog/libraries/website/javascript/querystring.md:56]:
```suggestion
> Currently, the `useQueryString` option is **only** available when you load analytics.js through the [NPM package](https://www.npmjs.com/package/@segment/analytics-next){:target="_blank"}.
```
```suggestion
> The `useQueryString` option is **only** available when you load analytics.js through the [NPM package](https://www.npmjs.com/package/@segment/analytics-next){:target="_blank"}.
```

File: src/connections/sources/catalog/libraries/website/javascript/querystring.md [hunk 7]
```diff
@@ -30,3 +30,27 @@ analytics.user().anonymousId('abc123');
...
+
+## How can I control query string processing?
+
+The `useQueryString` option allows you to control the behavior of the query parameters. For example, you can entirely disable query string processing by setting `useQueryString` to `false`:
+```js
+analytics.load('<WRITE_KEY>', {
+  useQueryString: false
+})
+```
+You can also keep query string processing on, but enforce validation rules. For example:
```
Comment (markzegarelli) [inline src/connections/sources/catalog/libraries/website/javascript/querystring.md:43]:
```suggestion

You can also keep query string processing on, but enforce validation rules. For example:

```

File: src/connections/sources/catalog/libraries/website/javascript/querystring.md [hunk 8]
```diff
@@ -30,3 +30,29 @@ analytics.user().anonymousId('abc123');
 ```
 
 Each trigger parameter is optional. You can pass up to **one of each trigger parameter** as shown in the example above.
+
+
+## How can I control query string processing?
+
+The `useQueryString` option allows you to control the behavior of the query parameters. For example, you can entirely disable query string processing by setting `useQueryString` to `false`:
```
Comment (markzegarelli) [inline src/connections/sources/catalog/libraries/website/javascript/querystring.md:37]:
```suggestion
The `useQueryString` option allows you to control the behavior of the query parameters. For example, you can entirely disable query string processing by setting `useQueryString` to `false`:

```

**Review by markzegarelli — APPROVED** [review]

**Pull request merged**